user.rb

```diff
@@ -22,20 +26,26 @@ def self.create(first_name:, last_name:, email:)
   end
 
   def self.write_new_user_to_db
```

Comment:
I wonder if there's a way that we could abstract away the persistence logic into a separate file (a module), that's not tied to users.

Reply:
What persistence logic do you mean? Everything involved in reading/writing to the file?